tests/test-dirs/locate/context-detection/mod_constr.t

```diff
@@ -1,9 +1,9 @@
 
-  $ $MERLIN single locate -look-for ml -position 2:13 -filename ./mod_constr.ml < ./mod_constr.ml
+  $ $MERLIN single locate -look-for ml -position 2:13 -filename ./mod_constr.ml < ./mod_constr.ml | sed 's/"[^""]*string.ml"/"irrelevant_path\/string.ml"/'
```

Comment:
I'm not fond of this.
Can't we fix merlin-wrapper instead? It's an issue that's bound to happen in other tests.

Reply:
Yes but what would be the correct behaviour ? Some path might be relevant top the test result, other not..